muikku-core-plugins/src/main/resources/META-INF/resources/scripts/gui/muikku-rich-memo-field-component.js

```diff
@@ -44,11 +44,15 @@
       this._editor.on('contentChange', $.proxy(this._onContentChange, this));
       this._editor.on('instanceReady', $.proxy(this._onInstanceReady, this));
       
-      // #3120 memo word counter (ui)
-      this._wordCountContainer = $('<div class="word-count-container">')
+      // #3120 memo word counter (ui) changed in #3757
+      this._countContainer = $('<div class="count-container">')
+        .append($('<span class="character-count-title">').text(getLocaleText('plugin.workspace.memoField.characterCount')))
```

Comment:
I suggest we add wrapper elements for word and character counter information so *-count-title  and *-count elements are easier to move around if necessary. The padding and margin is easier to adjusts then too.

Html output could be like this:

<div class="word-count-container">
  <span class="count-wrapper">
    <span class="character-count-title">...</span>
    <span class="character-count">...</span>
  </span>
  <span class="count-wrapper">
    <span class="word-count-title">...</span>
    <span class="word-count">...</span>
  </span>
</div>